inbox/models/when.py

```diff
@@ -97,7 +97,7 @@ def parse(cls, raw):
         # the singular form if they are equal.
         start, end = cls.parse_keys(raw)
         if start > end:
-            raise ValueError("'{}' must be < '{}'.".format(*cls.json_keys))
+            start, end = (end, start)
```

Comment:
Oh sorry it's here https://sentry.nylas.com/sentry/sync-prod/group/24197/.

We could also skip syncing the event and ignore it - totally up to you. I'm not sure that the `start < end` rule exists on the remote system though, I think we just made it up for ourselves when we built this `when` wrapper around time formats.